examples/gradle/dokka-multimodule-example/parentProject/build.gradle.kts

```diff
@@ -17,6 +18,19 @@ subprojects {
                 Visibility.PUBLIC,
                 Visibility.PROTECTED
             ))
+
+            // In multi-project builds, `remoteUrl` must point to that project's dir specifically, so if you
+            // want to configure sourceLinks at once in `subprojects {}`, you have to find the relative path.
+            // Alternatively, you can move this configuration up into subproject build scripts,
+            // and just hardcode the exact paths as demonstrated in the basic dokka-gradle-example.
```

Comment:
Could you add the link (https://kotlinlang.org/docs/dokka-gradle.html#source-link-configuration)?